convert-hf-to-gguf.py

```diff
@@ -2277,6 +2277,11 @@ def write_tensors(self):
         tensor_map = gguf.get_tensor_name_map(self.model_arch, block_count)
 
         for name, data_torch in self.get_tensors():
+            # lm_head is not used in llama.cpp, while autoawq will include this tensor in model
+            # To prevent errors, skip loading lm_head.weight.
+            if "lm_head.weight" in name:
+                    continue
```

Comment:
Let's change this to full string match and also print a short message that the tensor is being skipped

Reply:
already done in a new commit